modules/develop/pages/produce-data/configure-producers.adoc

```diff
@@ -232,6 +232,24 @@ verbose format, which impacts your compression speeds and network throughput.
 Alternatively, going with AVRO or Protobuf allows you to only define the schema
 in one place, while also enabling features like schema evolution.
 
+[[broker_timestamps]]
+=== Broker timestamps
+
+Redpanda employs a unique strategy to help ensure accuracy in retention operations. Every time a producer posts a message to a topic, Redpanda records the broker's system date and time in the message's `broker_timestamp` property. This helps ensure retention functions as intended in cases where a producer submits a message with a created timestamp skewed from the broker's time. Closed segments are only eligible for deletion under Redpanda's retention policy when the age of all messages in the segment exceed a set threshold. Using broker timestamps ensures that skewed messages are cleaned up appropriately where they may otherwise appear to never exceed the retention policy threshold.
+
+==== Configure broker timestamp alerting
+
+Redpanda increments the xref:reference:internal-metrics-reference.adoc#vectorized_kafka_rpc_produce_bad_create_time[`vectorized_kafka_rpc_produce_bad_create_time`] metric each time a broker receives a message with a skewed created timestamp that is outside a specified range. Two cluster properties control the threshold for this metric. The minimum accepted values for both of these properties is 5 minutes. Any attempt to set a value lower than that is rejected by Redpanda. The properties are:
+
+* `log_message_timestamp_alert_before_ms`: Defines the allowed skew before the broker's time. This check is effectively disabled when the value is set to `null`. Minimum: `300000 ms` (5 minutes), Default: `null`.
+* `log_message_timestamp_alert_after_ms`: Defines the allowed skew after the broker's time. There is no way to disable this check. Minimum: `300000 ms` (5 minutes), Default: `7200000 ms` (2 hours).
+
+==== Disable broker timestamp retention
+
+While not advised for typical use, Redpanda allows you to override the use of broker timestamps for retention policy. You accomplish this through use of the Admin API. Use the xref:api:ROOT:admin-api.adoc#Licenses-and-Features/operation/put_feature[`activate feature`] API to disable the `broker_time_based_retention` property.
+
+If you disable this feature, make sure to specify your desired timestamp policy. This is stored in the xref:reference:cluster-properties.adoc#log_message_timestamp_type[`log_message_timestamp_type`] cluster property. The timestamp policy defaults to `CreateTime` (client timestamp set by producer) but may be updated to `LogAppendTime` (server timestamp set by Redpanda).
```

Comment:
> The timestamp policy defaults to `CreateTime` (client timestamp set by producer) but may be updated to `LogAppendTime` (server timestamp set by Redpanda).

This is the opposite of what we seem to claim in the first sentence. Isn't the default set to the broker timestamp?

Reply:
The timestamp policy only comes into play if you disable broker timestamps -- in this specific scenario, disabling broker timestamps, the default becomes CreateTime.